larq_compute_engine/tests/end2end_test.py

```diff
@@ -192,5 +193,24 @@ def test_simple_model(model_cls):
     assert_model_output(model_lce, inputs, outputs)
 
 
+@pytest.mark.parametrize("model_cls", [toy_model, toy_model_int8])
+@pytest.mark.parametrize("inference_input_type", [tf.int8, tf.float32])
+@pytest.mark.parametrize("inference_output_type", [tf.int8, tf.float32])
+def test_int8_input_output(model_cls, inference_input_type, inference_output_type):
+    model_lce = convert_keras_model(
+        model_cls(),
+        inference_input_type=inference_input_type,
+        inference_output_type=inference_output_type,
+        experimental_default_int8_range=(-6.0, 6.0) if model_cls == toy_model else None,
+    )
+    interpreter = tf.lite.Interpreter(model_content=model_lce)
```

Comment:
I don't allocate any tensors here and just use the interpretor to get input and output types, thus we don't need to register our custom ops here.